docs/integrations/httpx.md

```diff
@@ -63,6 +63,152 @@ Let's see a minimal example below. You can run it with `python main.py`:
     asyncio.run(main())
     ```
 
+## Hooks
+
+The `logfire.instrument_httpx()` method also accepts the following hooks:
+
+- `request_hook`: called before sending a request.
+- `response_hook`: called after receiving a response.
+- `async_request_hook`: called before sending an async request.
+- `async_response_hook`: called after receiving an async response.
+
+You can use these hooks to add custom attributes that are not automatically captured by **Logfire**.
+
+### (Async) Request Hook
+
+The `request_hook` and `async_request_hook` hooks are called before sending a request. They receive two arguments:
+
+```py title="main.py" hl_lines="24-27"
```

Comment:
```suggestion
```py title="main.py"
```

Reply:
Why?

Reply:
Why are those lines highlighted? It looks weird. I thought it wasn't intentional.